### ISSUE TITLE
[Anon] Bug report: Imperium - Dark Angels.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#3675]
**File:** Imperium - Dark Angels.catz

**BattleScribe version:** 2.01.21

**Platform:** Android

**Dropbox:** No

**Description:** Characters for dark angels can not take relics unless made into the warlord. Relics do not become available when relics of the rock is chosen. Any character that is not named should have access to appropriate relics.

**Comment by Thairne** [comment]
As intended, You need a DA warlord to be able to take DA relics.

**Comment by Mad-Spy** [comment]
@Thairre, that is not correct. If you have a DA detachment, you unlock their stratagems. You can then take the Relics of the Rock stratagem to unlock relics without a Warlord.

**Comment by Thairne** [comment]
You can do that? Well hot diggedy damn.

**Comment by Mad-Spy** [comment]
Yup. It was in one of the FAQ, Death Guard I think.
